start-dev.sh

```
@@ -0,0 +1,226 @@
+VITE_API_URL=http://localhost:5001/api
+VITE_WS_URL=ws://localhost:5001
+VITE_APP_NAME=Smart Pen Monitor
+VITE_APP_VERSION=2.0.0
+VITE_ENVIRONMENT=development
+EOF
+fi
+
+# Function to start services with proper logging
+start_backend() {
+    echo -e "${BLUE}🔧 Starting backend server on port 5001...${NC}"
+    cd backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)